lib/rets/client.rb

```diff
@@ -9,11 +9,13 @@ class Client
 
     def initialize(options)
       @options = options
-      @capabilities = nil
       clean_setup
     end
 
     def clean_setup
+      if options.fetch(:login_after_error, false)
```

Comment:
@summera we had an mls complain to us about how many login attempts we were making. I'm quite possibly being overly cautious by adding this as an option but thought it might be useful for that mls